pysat/tests/test_meta.py

```diff
@@ -1999,33 +1363,22 @@ def teardown_method(self):
         del self.mutable
         return
 
-    # TODO(#789): remove tests for higher order meta
-    @pytest.mark.parametrize("prop,set_val", [('data', pds.DataFrame()),
-                                              ('ho_data', {})])
-    def test_meta_mutable_properties(self, prop, set_val):
-        """Test that @properties are always mutable.
-
-        Parameters
-        ----------
-        prop : str
-            Attribute on `self.meta` to be tested
-        set_val : any
-            Value to be assigned to `prop` on `self.meta`
-
-        """
+    def test_meta_mutable_properties(self):
+        """Test that @properties are always mutable."""
 
         # Set anything that can be immutable to be immutable
         self.meta.mutable = self.mutable
 
         # Test that data and label values can be updated
-        for prop, set_val in [('data', pds.DataFrame()), ('ho_data', {})]:
-            try:
-                # Pandas does not support dataframe equality
-                setattr(self.meta, prop, set_val)
-            except AttributeError:
-                raise AssertionError(
-                    "Couldn't update mutable property {:}".format(
-                        prop.__repr__()))
+        try:
+            # Pandas does not support dataframe equality
+            setattr(self.meta, 'data', pds.DataFrame())
+
+            # Test that data is empty
+            assert self.meta.data.empty, "`meta.data` not updated correctly"
```

Comment:
```suggestion
            assert self.meta.empty, "`meta.data` not updated correctly"
```

Updated test to increase coverage and also ensure `empty` is being updated.